test/python/test_pack.py

```diff
@@ -263,7 +263,18 @@ def test_packing_without_git(tmpdir):
         "--version", "0.1.0",
         project_path,
     ]
-    process = subprocess.run(cmd, cwd=tmpdir)
```

Comment:
I just realized that I missed `process.returncode == 0` assertion here.

The main idea of this test is to check that `cartridge pack` doesn't fail if application is not a git repository.

Reply:
Now, I check "process.returncode == 0"

But I don't see any matter to rename this test or split into several. It's not much and save time because "prepare step" is executed only once

Reply:
Your test is not about “packing application without git”. The “prepare step” just removes dependencies from project, it can be a fixture with session scope.

Reply:
Ok

Moved to `test_packing_with_version`